tests/integration/golang/admin/namespace/list_test.go

```diff
@@ -0,0 +1,52 @@
+package namespace
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/admin/api/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type ListNamespacesTestSuite struct {
+	helpers.BaseTestSuite
+}
+
+func TestListNamespacesTestSuite(t *testing.T) {
+	suite.Run(t, new(ListNamespacesTestSuite))
+}
+
+func (s *ListNamespacesTestSuite) Test_Ok() {
+	namespaces := map[string]*models.Namespace{}
+	defaultNamespace, err := s.NamespaceFixtures.GetDefaultNamespace(context.Background())
+	s.Require().Nil(err)
+	namespaces[fmt.Sprintf("%d", defaultNamespace.ID)] = defaultNamespace
```

Comment:
why do you need such a conversion? why just `namespaces := map[int]*models.Namespace{}` ? and then just `namespaces[defaultNamespace.ID]`?